tests/Lykke.AlgoStore.Tests/Unit/CodeBuildTests.cs

```diff
@@ -0,0 +1,182 @@
+        }
+
+        [Test]
```

Comment:
The following tests are essentially the same thing but with different code. Is there a need for them?